src/clj/tic_tac_toe_clojure/core.clj

```diff
@@ -0,0 +1,74 @@
+(ns tic-tac-toe-clojure.core (:require [clojure.string :as string]))
+(declare start-game)
+(defn -main [& args]
+  (println "Starting game")
+  (start-game))
+
+(defn create-board []
+  (into [] (repeat 9 "_")))
+
+(def numbered-board
+  ["0" "1" "2" "3" "4" "5" "6" "7" "8"])
+
+(defn build-board [board]
+  (with-out-str (println (string/join "\n" (re-seq #".{1,3}" (apply str board))))))
+
+(defn display-board [board]
+  (print (build-board board)))
+
+(defn get-user-symbol []
+  (do (print "Please choose a symbol (X or O): ") (flush) (read-line)))
+
+(defn get-user-position []
+ (Integer/parseInt (do (print "Please choose a position between 0 and 8: ") (flush) (read-line))))
+
+(defn set-position [board position user-symbol]
```

Comment:
Could this be `(def set-position assoc)`

Reply:
Wow that is so cool! Thanks for sharing!